ocaml/xapi/pvs_proxy_control.ml

```diff
@@ -249,6 +249,17 @@ let start_proxy ~__context vif proxy =
       | Api_errors.Server_error (code, args) when
           code = Api_errors.openvswitch_not_active ->
         "Host is not using openvswitch"
+      | Api_errors.Server_error ("SR_BACKEND_FAILURE_79", _) ->
```

Comment:
I wonder if we need to catch error 44 as well, just in case:

                <name>SRNoSpace</name>
                <description>There is insufficient space</description>
                <value>44</value>